hazelcast-sql/src/test/java/com/hazelcast/sql/support/expressions/ExpressionBiValue.java

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql.support.expressions;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+@SuppressWarnings({"unused", "unchecked", "checkstyle:MultipleVariableDeclarations"})
+public abstract class ExpressionBiValue extends ExpressionValue {
+    public static Class<? extends ExpressionBiValue> createBiClass(ExpressionType<?> type1, ExpressionType<?> type2) {
+        return createBiClass(type1.typeName(), type2.typeName());
+    }
+
+    public static Class<? extends ExpressionBiValue> createBiClass(String type1, String type2) {
+        try {
+            String className = ExpressionBiValue.class.getName() + "$" + type1 + type2 + "Val";
+
+            return (Class<? extends ExpressionBiValue>) Class.forName(className);
+        } catch (ReflectiveOperationException e) {
+            throw new RuntimeException("Cannot create " + ExpressionBiValue.class.getSimpleName() + " for types \""
+                + type1 + "\" and \"" + type2 + "\"", e);
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    public static <T extends ExpressionBiValue> T createBiValue(Class<? extends ExpressionBiValue> clazz) {
+        try {
+            return (T) clazz.newInstance();
+        } catch (ReflectiveOperationException e) {
+            throw new RuntimeException("Failed to create an instance of " + clazz.getSimpleName());
+        }
+    }
+
+    public static <T extends ExpressionBiValue> T createBiValue(
+        Class<? extends ExpressionBiValue> clazz,
+        int key,
+        Object field1,
+        Object field2
+    ) {
+        T res = create(clazz);
+
+        res.key = key;
+        res.field1(field1);
+        res.field2(field2);
+
+        return res;
+    }
```

Comment:
These methods are unused.
```suggestion
```

Reply:
These classes form a tiny framework for expression testing. In this specific PR, they are not used, but more usages will be added in the subsequent PRs with predicates and other expressions.